sub.mk

```diff
@@ -1,4 +1,5 @@
 CFG_FTPM_EMULATE_PPI ?= n
+CFG_TA_FTPM_RPMB_STORAGE ?= n
```

Comment:
Maybe the configuration should get the storage ID instead of a a boolean value that selects between `TEE_STORAGE_PRIVATE` and `TEE_STORAGE_PRIVATE_RPMB`. Here:
```makefile
CFG_FTPM_TA_TEE_STORAGE_ID ?= TEE_STORAGE_PRIVATE
```

Reply:
Sounds good.